### ISSUE TITLE
Sponsor page still says ebm donates the warehouse to us

### ISSUE THREAD

**Issue by gisellegk** [team178/team178.github.io#71]
Put it in past tense?

**Comment by TJScalzo** [comment]
I feel like we started with the fact that they donate the warehouse because it's an amazing, awesome thing for them to do. However, now that they don't do that anymore I feel like it'd be strange to leave the order the same and just change it to past tense.

**Comment by TJScalzo** [comment]
Here are some options. (I personally like the first and last one; the middle one sounds awkward.)

**With the sentence removed:**
>Headquartered in Farmington, CT, ebm-papst Inc. has given us several industrial class tools to build our robot. They assist us in the fabrication of robot parts and their engineers assist us in the design process. ebm-papst Inc. is the world’s leading source for engineered air movement solutions, provides a “total solution” approach to cooling requirements using extensive house resources. Every year, ebm-papst Inc. also donates thousands of fans that go inside speed controllers and other components as part of the annual FRC kit of parts to thousands of FIRST teams around the world. They are a “crown” level FRC supplier!

**With the sentence changed:**
>Headquartered in Farmington, CT, ebm-papst Inc. used to donate year-round use of a 3,000+ square foot warehouse in Unionville, CT that the Enforcers used to call home and has given us several industrial class tools to build our robot. They assist us in the fabrication of robot parts and their engineers assist us in the design process. ebm-papst Inc. is the world’s leading source for engineered air movement solutions, provides a “total solution” approach to cooling requirements using extensive house resources. Every year, ebm-papst Inc. also donates thousands of fans that go inside speed controllers and other components as part of the annual FRC kit of parts to thousands of FIRST teams around the world. They are a “crown” level FRC supplier!

**With the sentence changed and moved:**
>Headquartered in Farmington, CT, ebm-papst Inc. has given us several industrial class tools to build our robot. They assist us in the fabrication of robot parts and their engineers assist us in the design process. ebm-papst Inc. is the world’s leading source for engineered air movement solutions, provides a “total solution” approach to cooling requirements using extensive house resources. **They also used to donate year-round use of a 3,000+ square foot warehouse in Unionville, CT.** Every year, ebm-papst Inc. also donates thousands of fans that go inside speed controllers and other components as part of the annual FRC kit of parts to thousands of FIRST teams around the world. They are a “crown” level FRC supplier!

**Comment by gisellegk** [comment]
Maybe something like:

 "From (i don't remember when we got it) to 2015, they provided a 3,000+ square foot warehouse in Unionville, CT as a workspace for our team. Although we have since been moved out of the warehouse, we are still grateful for the hundreds (thousands? if you count Dream FIRST since it was developed there) of students who were inspired thanks to this generous donation" 

Either your short version or this longer one would fit best like how you moved it towards the end, I think.
